### ISSUE TITLE
Use HTML5 "number" type for numeric inputs.

### ISSUE THREAD

**Issue by thomas4019** [jdorn/json-editor#576]
I think when editing numeric values the input field should be the "number" type instead of "text". I can work on a pull request if this seems like a good change.

**Comment by DrPaulBrewer** [comment]
Using type number to enter one number is helpful on cell phones.  However, you also get the little up/down buttons which can be annoying but might (possibly?) be removed with styling.  

Entering multiple numbers could also be improved, with additional context.  For instance, a matrix or vector might be entered in a table.  I've been experimenting with using textarea for inputting a list of numbers for some research, and converting the string to an array of numbers with a bit of javascript,  For this, see https://github.com/jdorn/json-editor/issues/621

**Comment by thomas4019** [comment]
I agree about up and down spinner being annoying. But the extra validation
is nice and the fact that it limits most letters.

It looks like you can remove the spinner using CSS.

https://stackoverflow.com/questions/23372903/hide-spinner-in-input-number-firefox-29

The textarea idea for multivalued fields sounds interesting too.
